index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Atleti</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" type="text/css" href="//https://fonts.googleapis.com/css2?family=Outfit" />
+</head>
+<header class="main-header">
+
+    <div class="logo">
+        <img src="atletilogo.png" width="200" height="200" alt="AtletiLOGO">
+ 
+        <div class="sitename">
+            <p class="h1">Atleti.</p>
+        <p class="h2"><em>The number one source for the latest tennis news</em></p>
```

Comment:
Suggestion here, I don't think you need to wrap the `h1` and `h2` elements in a paragraph element. You could instead do a `<h1 class="...">.` Or with no class if not needed.